### ISSUE TITLE
bug with age_prior_model in compute_age_grid_weights()

### ISSUE THREAD

**Issue by lea-hagen** [BEAST-Fitting/beast#426]
I'm creating a physics model grid, and it gave me this error.  Maybe it's a bug, or maybe there's been a relevant PR since I last updated (which was sometime yesterday).

```
/astro/dust_kg3/lhagen/stsci/beast_lea-hagen/beast/tools/run/create_physicsmodel.py in create_physicsmodel(nsubs, nprocs, subset)
    110         age_prior_model=datamodel.age_prior_model,
    111         mass_prior_model=datamodel.mass_prior_model,
--> 112         met_prior_model=datamodel.met_prior_model,
    113     )
    114

/astro/dust_kg3/lhagen/stsci/beast_lea-hagen/beast/physicsmodel/model_grid.py in add_stellar_priors(project, specgrid, verbose, priors_fname, **kwargs)
    273         full filename to which to save the spectral grid with priors
    274
--> 275     Returns
    276     -------
    277     fname: str

/astro/dust_kg3/lhagen/stsci/beast_lea-hagen/beast/physicsmodel/grid_and_prior_weights.py in compute_age_mass_metallicity_weights(_tgrid, **kwargs)
     59     # setup the vector to hold the z weight vector
     60     total_z_grid_weight = np.zeros(len(uniq_Zs))
---> 61     total_z_prior_weight = np.zeros(len(uniq_Zs))
     62     total_z_weight = np.zeros(len(uniq_Zs))
     63

TypeError: compute_age_grid_weights() got an unexpected keyword argument 'age_prior_model'
```

**Comment by karllark** [comment]
I haven't seen that error before.  Hum....

**Comment by karllark** [comment]
That will be fixed with PR #423.  I'll review that PR now so that it can be merged as soon as possible.

**Comment by lea-hagen** [comment]
awesome, thanks!
